README.md

```diff
@@ -24,36 +28,29 @@ This SDK is a wrapper around different platforms such as JVM, Android, iOS, macO
 
 ## Configure Repository
 
-The Kotlin Multiplatform SDK is available only on `mavenLocal`. You can declare this repository in your build script as follows:
+The Kotlin Multiplatform SDK is available on `mavenCentral`. You can declare this repository in your build script as follows:
 
 ```gradle
 repositories {
-  // Currently only supported locally
-  mavenLocal()
+  mavenCentral()
 }
 ```
 
-and then run:
-
-```bash
-./gradlew publishToMavenLocal
-```
-
 ## Add dependency
 For a multiplatform project, you need to add the sentry-kotlin-multiplatform artifact to the `commonMain` source set:
 
 ```Kotlin
 val commonMain by getting {
   dependencies {
-    api("io.sentry:sentry-kotlin-multiplatform:0.0.1")
+    api("io.sentry:sentry-kotlin-multiplatform:0.0.1-alpha.2")
   }
 }
 ```
 
 ### Cocoa
 
 If you are targeting Apple platforms (iOS, macOS, watchOS, tvOS), then you need to use CocoaPods to include [Sentry Cocoa](https://github.com/getsentry/sentry-cocoa) into this SDK.
-One way to achieve this is to include the Sentry Cocoa SDK via the Kotlin CocoaPods extension. Be aware that your Sentry Cocoa version has to match the SDK's version. Currently the supported version is `~> 7.21.0` 
+One way to achieve this is to include the Sentry Cocoa SDK via the Kotlin CocoaPods extension. Be aware that your Sentry Cocoa version has to match the SDK's version. Currently the supported version is `pod("Sentry", "~> 8.2.0")` 
```

Comment:
Is there a way to do this automatically though? so People install the KMP SDK and that pulls the compatible version of the Cocoa SDK?

Reply:
Not yet unfortunately. Users need to pull in the cocoa sdk dep themselves as KMP afaik only includes the headers in their apple packages so the actual implementation of these headers has to be provided from somewhere else. 

that also means that users aren't necessarily tied to using cocoapods. but I haven't tested yet if SPM or Carthage works (theoretically should though).